contributors.json

```diff
@@ -230,5 +230,11 @@
   "xxx32": {
     "country": "India",
     "name": "Aarushi"
+  },
+  "femakin": {
```

Comment:
One last thing to do is order this in ascending order so that there won't be many conflicts with other PRs